### PR TITLE
Redesigned wallet connect page

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: src/Modules/NetworkUnsupportedModal.tsx [hunk 1]
```diff
@@ -21,10 +21,10 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
     inner: {
       width: "100% !important",
       maxWidth: "unset !important",
-      borderRadius: "4px",
+      borderRadius: 5,
       display: "flex",
       flexDirection: "row",
-      padding: `${constants.generalUnit * 6}px ${constants.generalUnit * 7}px`,
+      padding: `${constants.generalUnit * 3}px ${constants.generalUnit * 3}px`,
```
Comment (AndreiNavoichyk) [inline src/Modules/NetworkUnsupportedModal.tsx:27]: Remove `px`

File: src/Components/Pages/TransferPage.tsx [hunk 2]
```diff
@@ -48,9 +51,55 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
...
       fontFamily: styles.primaryFont,
-      padding: constants.generalUnit * 3,
+      padding: constants.generalUnit * 2.5,
       position: "relative",
       backgroundColor: "white",
+      minWidth: 460,
+    },
+    wrapper: {
+      position: "relative",
+      background: "#FFFFFF",
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: Apply the formatting agreements we've agreed with you on in prev PRs.

File: src/Modules/NetworkUnsupportedModal.tsx (hunk 1 above)
Comment (karthikeyan-cere) [reply src/Modules/NetworkUnsupportedModal.tsx:27]: Need to mention the measuring units for shorthand properties.

File: src/Components/Pages/TransferPage.tsx (hunk 2 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Refactored

File: src/Components/Pages/TransferPage.tsx [hunk 3]
```diff
@@ -48,9 +51,55 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
...
+    },
+    headerText: {
+      fontWeight: "bold",
+      fontSize: 30,
+      lineHeight: `${constants.generalUnit * 3}px`,
+      color: "black",
+      paddingLeft: 12,
+    },
+    selectArea: {
+      padding: `${constants.generalUnit * 2.5}px ${
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx:81]: remove `px`

File: src/Components/Pages/TransferPage.tsx [hunk 4]
```diff
@@ -75,12 +124,47 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
...
+      textAlign: "center",
+    },
+    EthWalletBtn: {
+      width: "100%",
+      fontSize: 14,
+      lineHeight: `${constants.generalUnit * 6}px`,
+      fontWeight: "bold",
+      background: "linear-gradient(105.79deg, #6300E1 1.84%, #DD00E1 92.41%)",
+      borderRadius: 5,
+      color: "#FFFFFF",
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: use name

File: src/Components/Pages/TransferPage.tsx [hunk 5]
```diff
@@ -101,7 +185,7 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
       borderRadius: 2,
       color: palette.additional["gray"][9],
       marginTop: constants.generalUnit,
-      fontSize: 11,
+      fontSize: 12,
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx:188]: Confirm this size work with long addresses.

File: src/Components/Pages/TransferPage.tsx [hunk 6]
```diff
@@ -411,231 +497,263 @@ const TransferPage = () => {
...
+        ) : (
+          <>
+            <section className={classes.connected}>
+              <div>
+                <div className={classes.inputLabel}>Home Network</div>
+                <Typography
+                  className={classes.changeButton}
+                  variant="body1"
+                  onClick={async () => {
+                    await Promise.all([
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx:549]: Have you reused this logic or implemented it from scratch?

File: src/Components/Pages/TransferPage.tsx (hunk 4 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: Refactored.

File: src/Components/Pages/TransferPage.tsx (hunk 6 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx:549]: Implemented from scratch.

File: src/Components/Pages/TransferPage.tsx (hunk 3 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx:81]: Need to mention the measuring units for shorthand properties.

File: src/Components/Pages/TransferPage.tsx (hunk 5 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx:188]: Confirmed font size 12 work with long addresses.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**